app/main.py

```diff
@@ -1,6 +1,14 @@
+import requests
+
+from pprint import pprint
+
+import os
+
+
 def get_weather():
-    # write your code here
-    pass
+    api_key = os.environ.get("API_KEY")
+    request_data = requests.get(f"https://api.weatherapi.com/v1/current.json?key={api_key}&q=Paris").json()
+    pprint(request_data)
```

Comment:
you shouldn't print row data from API, choose useful information and make the output look good. You have an example on the image